packages/amplify-graphql-auth-transformer/src/graphql-auth-transformer.ts

```diff
@@ -20,23 +22,27 @@ import {
   DataSourceProvider,
   MutationFieldType,
   QueryFieldType,
-  TransformerTransformSchemaStepContextProvider,
+  TransformerAuthProvider,
+  TransformerBeforeStepContextProvider,
   TransformerContextProvider,
+  TransformerPreProcessContextProvider,
```

Comment:
This appears to be unused.
```suggestion
```

Reply:
D'oh! Thanks for catching. removed in 4446c34c6